src/mockatoo/macro/MockMaker.hx

```diff
@@ -167,9 +167,9 @@ class MockMaker
 	*/
 	function createMockFromStruct(fields:Array<ClassField>)
 	{
-		var args:Array<{ field : String, expr : Expr }> = [];
+		var args:Array<ObjectField> = [];
```

Comment:
You may wish to `import tink.MacroApi.ObjectField;` if you care about haxe3 backward compatibility (as suggested by other changes).